packages/react-client/src/useTelnyxClient.ts

```diff
@@ -1,52 +1,34 @@
-import { useState, useRef, useEffect } from 'react';
+import { useEffect, useRef } from 'react';
 import { TelnyxRTC } from '@telnyx/webrtc';
 
-enum ClientState {
-  READY = 'ready',
-  ERROR = 'error',
-  DISCONNECTED = 'disconnected',
-}
-
-interface IPartialCall {
-  state: string; // TODO typings for call state
-  direction: 'inbound' | 'outbound';
-  options: {
-    remoteCallerName: string;
-    remoteCallerNumber: string;
-    destinationNumber: string;
-    telnyxCallControlId: string;
-  };
-  answer: Function;
-  hangup: Function;
-  remoteStream?: MediaStream;
-  muteAudio: Function;
-  unmuteAudio: Function;
-}
-
-type TokenCredentialOptions = {
+type TokenCredential = {
   login_token: string;
 };
 
-type UsernameCredentialOptions = {
+type UsernameCredential = {
   login: string;
   password: string;
 };
 
-type CredentialOptions = TokenCredentialOptions | UsernameCredentialOptions;
+export type CredentialOptions = TokenCredential | UsernameCredential;
 
 /**
- * Constructs a Telnyx client, connects the client and subscribes
- * to events, e.g. an event for an incoming call.
+ * Constructs a Telnyx client, connects the client, and handles
+ * disconnecting the client during cleanup.
  *
  * ## Examples
  *
  * import { useTelnyxClient } from '@telnyx/react-client'
  *
  * // Login using On-Demand Credentials token
- * const { client, call, clientState } = useTelnyxClient({ login_token })
+ * const client = useTelnyxClient({ login_token })
  *
  * // Or, login using your SIP Connection username and password
- * // const { client, clientState } = useTelnyxClient({ login, password })
+ * // const client = useTelnyxClient({ login, password })
+ *
+ * client.on('telnyx.notification', ({ call }) => {
+ *   console.log(call)
+ * })
```

Comment:
If you add this between backticks, VS Code will highlight the code just like Markdown